syncopy/tests/test_preproc.py

```diff
@@ -138,26 +139,40 @@ def test_but_kwargs(self):
                       'order': 4}
             # only for firws
             if 'minphase' in direction:
+
```

Comment:
I think here the pytest-native fail-assertion would be more elegant than the `failed` try/except block:
```python
with pytest.raises(SPYValueError) as err:
   self.test_but_filter(**kwargs)
   assert "expected 'onepass'" in str(err)
```
This fails if `self.test_but_filter(**kwargs)` does not raise an exception, hence no need to incorporate `failed = True/False`

Reply:
Same for the remaining `failed = True` tests

Reply:
aah, yeah that's clearly the better way 👍